library/L0_Platform/stm32f10x/linker.ld

```diff
@@ -0,0 +1,181 @@
+MEMORY
+{
+    /* Flash memory region for the program, offset by the bootloader section */
+    FLASH (rx) : ORIGIN = 0x08000000, LENGTH = 64k
+
+    /* 32K (Heap starts here) */
+    SRAM (rwx) : ORIGIN = 0x20000000, LENGTH = 20k
+
+    /* 32K (Globals at bottom, stack on top), heap continues from here */
+    /* SRAM (rwx) : ORIGIN = 0x2007c000, LENGTH = 32K */
+
+    /* Heap starts at SRAM because if heap starts from higher memory, then
+     * malloc tracking routines get confused when sbrk() system function
+     * returns higher heap memory first, and then lower heap memory.
+     * So we start heap from SRAM, and then move it up to SRAM if SRAM
+     * portion runs out completely.
+     */
+
+     /* Symbol Table */
+     /*FLASH_SYMBOL_TABLE : ORIGIN = 0x1000, LENGTH = 32K*/
+}
+
+  /* Define a symbol for the top of each memory region */
+  __base_MFlash512 = 0x0  ; /* MFlash512 */
+  __base_Flash = 0x0 ; /* Flash */
+  __top_MFlash512 = 0x0 + 0x80000 ; /* 512K bytes */
+  __top_Flash = 0x0 + 0x80000 ; /* 512K bytes */
+  __base_RamLoc64 = 0x10000000  ; /* RamLoc64 */
+  __base_RAM = 0x10000000 ; /* RAM */
+  __top_RamLoc64 = 0x10000000 + 32K ; /* 64K bytes */
+  __top_RAM = 0x10000000 + 32K ; /* 64K bytes */
+  __base_RamPeriph32 = 0x2007c000  ; /* RamPeriph32 */
+  __base_RAM2 = 0x2007c000 ; /* RAM2 */
+  __top_RamPeriph32 = 0x2007c000 + 0x8000 ; /* 32K bytes */
+  __top_RAM2 = 0x2007c000 + 0x8000 ; /* 32K bytes */
+  __user_heap_base = __base_RAM2 ; /* SJSU-Dev2: RAM2 is solely for heap */
+
+/* Program entry point */
+ENTRY(ArmResetHandler)
+
+SECTIONS
+{
+	/* MAIN TEXT SECTION */
+	.text : ALIGN(4)
+	{
+		FILL(0xff)
+		KEEP(*(.isr_vector))
+
+		/* Global Section Table */
+		. = ALIGN(4) ;
+		section_table_start = .;
+		data_section_table = .;
+		LONG(LOADADDR(.data));
+		LONG(    ADDR(.data));
+		LONG(  SIZEOF(.data));
+		data_section_table_end = .;
+		bss_section_table = .;
+		LONG(    ADDR(.bss));
+		LONG(  SIZEOF(.bss));
+		bss_section_table_end = .;
+		section_table_end = . ;
+		/* End of Global Section Table */
+
+        /* Functions that are placed after the interrupt vector */

```

Comment:
Fix indentation throughout this whole. Must use spaces not tabs.

Reply:
This has not been resolved.

Reply:
Good catch, I accidentally selected the VS code that only starts using spaces instead of tabs, instead of converting everything. Should be fixed now.